src/doc/rustc/src/platform-support/nto-qnx.md

```diff
@@ -0,0 +1,117 @@
+# nto-qnx
+
+**Tier: 3**
+
+[BlackBerry® QNX®](https://blackberry.qnx.com/) Neutrino (nto) Real-time operating system.
```

Comment:
Perhaps we should have a discussion about this.

-- QNX.

Reply:
Should be fixed now.